using-mobile-extensions/adobe-campaign-standard-beta.md

```diff
@@ -46,62 +43,60 @@ Need help connecting your Launch mobile property with your Campaign Standard mob
 Trying to find your MCIAS endpoint URL? Contact your beta manager.
 {% endhint %}
 
-Provide your in-app messaging service URL endpoint. The SDK retrieves all in-app messaging rules and definition payloads from this endpoint.
+Select an MCIAS endpoint based on the region where your customer is located or enter a custom endpoint. The SDK retrieves all in-app messaging rules and definition payloads from this endpoint.
 
 {% hint style="warning" %}
-For this beta extension, the MCIAS endpoint URL should be provided **with** a prefixing`http://` or `https://` and **must** end with a forward slash. For example, `https://mcias-va7.cloud.adobe.io/mcias/`.
+For this extension, the MCIAS endpoint URL should be provided **without** a prefixing`http://` or `https://` and **must not** end with a forward slash. For example, `mcias-va7.cloud.adobe.io/mcias`.
```

Comment:
Please change the sentence to read as follows:

"For this extension, the MCIAS endpoint URL should be provided **without** the `http://` or `https://` and **must not** end with a forward slash. "